hedera-node/hedera-mono-service/src/main/java/com/hedera/services/throttling/DeterministicThrottling.java

```diff
@@ -708,7 +712,7 @@ private ChildTransactionDetails(
         public int getNumAutoCreations() {
```

Comment:
_Nit_ - maybe rename to `getNumImplicitCreations()`?

Reply:
Since it will include the lazy creations as well I agree it might be better to rename. Did you mean renaming only this method or everything related to auto creations e.g. in `TxnAccessor` as well?

Reply:
Renamed from autoCreations to implicitCreations where both auto & lazy creations are present.